packages/connect-explorer-nextra/src/components/fields/File.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button } from '@trezor/components';
+
+import { Row } from './Row';
+import type { Field } from '../../types';
+
+interface FileProps {
+    field: Field<any>;
+    disabled?: boolean;
+    onChange: (field: Field<any>, value: any) => any;
+}
+
+const File = ({ disabled, field, onChange }: FileProps) => {
+    const onFilesAdded = evt => {
```

Comment:
I know this is copied from old code, but since we are here why don't we call it `event`? and the one bellow `onLoadEvent`?

Reply:
Yes, this is just copied and unchanged for now